content/en/docs/pipelines/pipelines-quickstart.md

```diff
@@ -3,12 +3,6 @@ title = "Pipelines Quickstart"
 description = "Getting started with Kubeflow Pipelines"
 weight = 10
                     
-+++
```

Comment:
You shouldn't remove this line, it causes presubmit failure.

Reply:
opss damn.. fixed !